mottak/src/main/kotlin/no/nav/melosys/soknadmottak/mottak/MottakService.kt

```diff
@@ -98,18 +94,61 @@ class MottakService(
         }
     }
 
+    @Scheduled(fixedDelay = VENTETID_MELLOM_JOBBER_MILLIS * 12, initialDelay = OPPSTART_FØRSTE_JOBB_MILLIS)
+    fun lagSøknadPdfHvisManglerEtterFeil() {
+        try {
+            withLoggingContext(MDC_CALL_ID to UUID.randomUUID().toString()) {
+                soknadService.hentIkkeLeverteSøknader()
+                    .filter { !finnesPåDownloadQueue(it.arkivReferanse) }
+                    .filter { manglerBareSøknadPdf(it) }
```

Comment:
Sliter litt med navnet på denne, men det kan hende det bare er domenekunnskap som mangler. Du skal ikke lage søknad dersom ikke-levert-søknad kun mangler pdf?

Reply:
Jeg er enig i at navnet her skurrer litt. Kan man ikke lage "harSøknadPdf"? Såvidt jeg kan se er det alt metoden sjekker for

Reply:
Man skal lage pdf dersom ikke-levert-søknad kun mangler pdf

Hvordan kan jeg uttrykke det bedre?

Reply:
Ja, kan endre det til `!harSøknadPdf` f.eks.

Reply:
Hvis det blir rett logisk sett, blir det nok lettere å lese det 👍